src/patternfly/components/Title/docs/code.md

```diff
@@ -0,0 +1,16 @@
+## Overview
+When you don't want to use the Content component and just want to use single title styles.
```

Comment:
Lets just say what the component does and then reference the content component for larger blocks of text.
Something like, "the title components changes the size of headers" or whatever.